gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -188,9 +192,20 @@
                         For more information on inherited controls for containers see the DISA
                         Container Hardening Process Guide.</ns0:rationale>
                     <ns0:platform idref="cpe:/o:notapplicable" />
+                    <ns0:ident system="http://cyber.mil/legacy">V-263651</ns0:ident>
+                    <ns0:ident system="http://cyber.mil/legacy">SV-263651</ns0:ident>
+                    <ns0:ident system="http://cyber.mil/cci">CCI-000366</ns0:ident>
```

Comment:
The same CCI (`CCI-000366`) is used throughout. They are different per Vuln/Rule; for example, the one for `V-263651` here is `CCI-003959` (https://stigaview.com/products/srg-gpos/v3r2/SRG-OS-000690-GPOS-00140/)

There's a whole lookup table on what CCI mapping are: https://public.cyber.mil/stigs/cci/

Reply:
Fixed in: [`35c8b42` (#20)](https://github.com/chainguard-dev/stigs/pull/20/commits/35c8b42d08dabf4f04c912a8a698ee9fd7a423d6)